src/main/java/org/reaktivity/nukleus/tls/internal/TlsNukleus.java

```diff
@@ -125,9 +128,8 @@ private boolean handleRoute(
             case RouteFW.TYPE_ID:
             {
                 final RouteFW route = routeRO.wrap(buffer, index, index + length);
-                handleRoute(route);
+                return handleRoute(route);
```

Comment:
Let's keep this as single `return` with `boolean handled = false;` at the top and `handled = handleRoute(...);` here.